### PR TITLE
Describe test plans for coverage of Qute

### PR DESCRIPTION
**Pull request by fedinskiy**
Relevant issues:
https://issues.redhat.com/browse/QUARKUS-2328
https://issues.redhat.com/browse/QUARKUS-2329
https://issues.redhat.com/browse/QUARKUS-2330

### CODE REVIEW

File: QUARKUS-2330.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- Bare metal
+  - 1 minute for JVM mode
+  - 4 minute for native mode
+- OpenShift
+  - 6 minutes for JVM mode
+  - 12 minutes for native mode
+
+Additionally, we need to enable qute module in quickstarts(<1 minute running time in JVM mode) for our acceptance and interoperability tests.
+
+## Automated test development
```
Comment (rsvoboda) [inline QUARKUS-2330.md:39]: What's the current estimation for TD?

**Review by rsvoboda — CHANGES_REQUESTED** [review]

File: QUARKUS-2330.md (hunk 1 above)
Comment (fedinskiy) [reply QUARKUS-2330.md:49]: 5-10 days. Added time estimation.

File: QUARKUS-2330.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+9. Encoding: non-latin, RTL and multibyte scripts (this is not covered)
+10. Unparsed Character Data (covered upstream but not for multiline)
+11. User-defined tags (covered)
+12. Values resolvers (covered)
+13. Content filters (covered)
+14. Direct bean injection (covered) with type-safe Expressions (covered)
+15. @CheckedTemplate (covered)
+16. Build-name templates: maps (not covered), lists (covered), integers (covered), strings (covered), time (covered)
+17. Other templates: @TemplateData (covered), @TemplateEnum (covered), @TemplateGlobal (covered), @CheckedTemplate (covered). These are not expected to work in native mode, though
+18. Content negotiation.
```
Comment (rsvoboda) [inline QUARKUS-2330.md:34]:
```suggestion
18. Content negotiation
```

File: QUARKUS-2330.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+   1. rendering of tags (parsed and unparsed covered)
+   2. expressions (4 types, coverage status unclear)
+   3. arrays (covered partially)
+   4. loops (covered partially, especially metadata),
+   5. character escapes and raw data (covered),
+   6. virtual methods (covered)
+   7. `if` and `when` sections (covered)
+   8. template inheritance (covered, probably worth some addition)
+   9. with (covered) and let (covered) sections
+   10. Eval sections (covered).
```
Comment (rsvoboda) [inline QUARKUS-2330.md:24]:
```suggestion
   10. eval sections (covered)
```

File: QUARKUS-2330.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+9. Encoding: non-latin, RTL and multibyte scripts (this is not covered)
+10. Unparsed Character Data (covered upstream but not for multiline)
+11. User-defined tags (covered)
+12. Values resolvers (covered)
+13. Content filters (covered)
+14. Direct bean injection (covered) with type-safe Expressions (covered)
+15. @CheckedTemplate (covered)
+16. Build-name templates: maps (not covered), lists (covered), integers (covered), strings (covered), time (covered)
+17. Other templates: @TemplateData (covered), @TemplateEnum (covered), @TemplateGlobal (covered), @CheckedTemplate (covered). These are not expected to work in native mode, though
```
Comment (rsvoboda) [inline QUARKUS-2330.md:33]: > These are not expected to work in native mode, though

Where is this info tracked?

Comment (fedinskiy) [reply QUARKUS-2330.md]: Changed the wording.

**Review by mjurc — APPROVED** [review]
This looks like a good plan to me now, thanks.

**Pull request merged**